jasmine.js

```diff
@@ -0,0 +1,23 @@
+describe("firstTest", function() {
+
+	it("check result", function() {
+      		expect(tests.firstTest()).toEqual(30)
+	});
+
+});
+
+describe("secondTest", function() {
+
+	it("check string result", function() {
+    		expect(tests.secondTest()).toEqual(jasmine.stringMatching('11 & 10'));
+	});
+
+});
+
+describe("thirdTest", function() {
+
+	it("check string result", function() {
+    		expect(tests.secondTest()).toEqual(jasmine.stringMatching('4 5 10'));
```

Comment:
the third problem requires you to determine if an array has a duplicates or not, there is no requirement to return an array